prometheus-ksonnet/images.libsonnet

```diff
@@ -1,12 +1,12 @@
 {
   _images+:: {
-    prometheus: 'prom/prometheus:v2.2.1',
+    prometheus: 'prom/prometheus:v2.3.1',
     watch: 'weaveworks/watch:master-5b2a6e5',
-    kubeStateMetrics: 'gcr.io/google_containers/kube-state-metrics:v1.2.0',
-    grafana: 'grafana/grafana:5.1.3',
+    kubeStateMetrics: 'gcr.io/google_containers/kube-state-metrics:v1.3.1',
+    grafana: 'grafana/grafana:5.2.1',
     gfdatasource: 'quay.io/weaveworks/gfdatasource:master-2bda599',
-    alertmanager: 'prom/alertmanager:v0.14.0',
-    nodeExporter: 'prom/node-exporter:v0.15.2',
+    alertmanager: 'prom/alertmanager:v0.15.0',
+    nodeExporter: 'prom/node-exporter:v0.16.0',
```

Comment:
This release breaks everything, the node exporter mixin needs a lot of work.  So I'd avoid it for now.

Reply:
Done